encoding/protobuf/protoc-gen-yarpc-go/internal/lib/lib.go

```diff
@@ -128,6 +218,43 @@ func (c *_{{$service.GetName}}YARPCCaller) {{$method.GetName}}(ctx context.Conte
 	return c.client.CallOneway(ctx, "{{$method.GetName}}", request, options...)
 }
 {{end}}
+{{range $method := clientStreamingMethods $service}}
+func (c *_{{$service.GetName}}YARPCCaller) {{$method.GetName}}(ctx context.Context, options ...yarpc.CallOption) ({{$service.GetName}}Service{{$method.GetName}}YARPCClient, error) {
+	stream, err := c.client.CallStream(ctx, "{{$method.GetName}}", options...)
+	if err != nil {
+		return nil, err
+	}
+	return &_{{$service.GetName}}Service{{$method.GetName}}YARPCClient{stream: stream}, nil
+}
+{{end}}
+{{range $method := serverStreamingMethods $service}}
+func (c *_{{$service.GetName}}YARPCCaller) {{$method.GetName}}(ctx context.Context, request *{{$method.RequestType.GoType $packagePath}}, options ...yarpc.CallOption) ({{$service.GetName}}Service{{$method.GetName}}YARPCClient, error) {
+	stream, err := c.client.CallStream(ctx, "{{$method.GetName}}", options...)
+	if err != nil {
+		return nil, err
+	}
+	reader, closer, err := protobuf.ToReader(request, stream.Request().Encoding)
```

Comment:
oh, now I see where this is

this function needs a better name, and if public, should be in `encoding/protobuf/protobuf.go`